### DISCUSSION TITLE
"HTTP/2 stream 1 was reset"

### DISCUSSION THREAD

**Discussion by atoff** [getsentry/sentry-laravel#703] [Q&A] [answered]
Hello,

I've had the bug similar to https://github.com/getsentry/sentry-php/issues/1537, whereby sentry events are not sent because of the stream being reset. 
```
[sentry-laravel] error: Failed to send the event to Sentry. Reason: "HTTP/2 stream 1 was reset for ...
```

I see this has been fixed in https://github.com/getsentry/sentry-php/releases/tag/3.19.1 for the PHP SDK, but I don't think this has been ported into this SDK here - just wanted to check if this was already in the works before I put an issue in!

Thanks :)

**Comment by cleptric** [comment, marked as answer]
You can run `composer update sentry/sentry-laravel -W`. The Laravel SDK has the PHP SDK as its [dependency](https://github.com/getsentry/sentry-laravel/blob/master/composer.json#L26).

**Reply by atoff** [reply]
Oh sorry - that makes sense! I think the package name differing from the repository threw me off slightly. Thanks!
